src/PartOfMultipartStream.php

```diff
@@ -98,27 +87,27 @@ public function rewind(): void
         $this->stream->rewind();
     }
 
-    public function isWritable()
+    public function isWritable(): bool
     {
         return $this->stream->isWritable();
     }
 
-    public function write($string)
+    public function write($string): int
```

Comment:
Hm, we should have proper type-hints here:

https://github.com/php-fig/http-message/blob/402d35bcb92c70c026d1a6a9883f06b2ead23d71/src/StreamInterface.php#L115